app/controllers/whitehall_media_controller.rb

```diff
@@ -1,56 +1,31 @@
-class WhitehallMediaController < BaseMediaController
```

Comment:
Can we get rid of `BaseMediaController` now? Perhaps merge it into `MediaController`?